### ISSUE TITLE
ocp_installer.sh should hint the ip is private ip

### ISSUE THREAD

**Issue by longdafeng** [oceanbase/oceanbase#745]
in the configuration

```
OCP_IP_ARRAY=()     # （IP1 IP2 IP3）

SSH_USER=root    # if not root, make sure remote user can use sudo without password    admin   ALL=(ALL)   NOPASSWD:ALL
SSH_PORT=22
SSH_AUTH=pubkey     # can be password or pubkey
SSH_PASSWORD=''     # password for passowrd auth, when use pubkey auth, is passphrase
SSH_KEY_FILE='/root/.ssh/id_rsa'   # pubkey auth
```

in this part, 
```
OCP_IP_ARRAY=()     # （IP1 IP2 IP3）
```
it should suggest use private ip for OCP_IP_ARRAY.

In aliyun cloud, if use public ip here, the scp speed is so slow that use misunderstand the installation is dead.

**Comment by chris-sun-star** [comment]
the install script will use the IP address to copy install files, when using public ip, the speed is really slow
after install, the IP address will be part of config-server address which will be set to observer
provide private ip is ok

**Comment by chris-sun-star** [comment]
fixed in 3.1.1-ce-bp1
